tests/framework/clients/object.go

```diff
@@ -90,6 +90,49 @@ spec:
 	return k8sh.CreateExternalRGWService(namespace, storeName)
 }
 
+func (ro *ObjectOperation) ObjectDelete(namespace, storeName string, replicaCount int32, callAPI bool, k8sh *utils.K8sHelper) error {
+	store := model.ObjectStore{Name: storeName, Gateway: model.Gateway{Instances: replicaCount, Port: model.RGWPort}}
+	store.DataConfig.ReplicatedConfig.Size = 1
+	store.MetadataConfig.ReplicatedConfig.Size = 1
+	if callAPI {
```

Comment:
this looks duplicated from the smoke tests. Could it be refactored? I would envision the tests would be written in a single place, then the test suites could define which tests they run. Then for example both the longhaul and smoke suites could run a test without any code duplication.

Reply:
these are helper functions, implementation of object interface. Both longhaul and integration test use the same library.  The Create and Delete ObjectStore look very similar( its the same case with create and delete of file and object store interface).

Reply:
i see, this is delete. Should the smoke test be calling this during the test teardown as well?

Reply:
yes, i will add it to the smoke tet as well.